data/json/vitamin.json

```diff
@@ -3,39 +3,64 @@
     "id": "calcium",
     "type": "vitamin",
     "name": "Calcium",
-    "min": -600,
-    "deficiency": [ [ "brittlebone", -300 ] ]
+    "deficiency": "hypocalcemia",
+    "min": -720,
+    "disease": [
+      [ -288, -336 ],
+      [ -337, -384 ],
+      [ -385, -720 ]
```

Comment:
Yes, but until the design is finalised I don't want to rework the JSON again as part of this PR.

To put it into context there are only five vitamin ranges to specify so whilst such optimisations aren't incorrect they aren't sufficiently a priority to trigger at least another round of code review especially given the design of the JSON could quite conceivably be further changed.